### ISSUE TITLE
access buttons in the screenlocker

### ISSUE THREAD

**Issue by drzraf** [souch/SMP#13]
My phone is often locked (auto-lock after some time).
It would be very useful to have a quick access to most useful features while locked.
The default player of Android 4.4 provides this feature.

Thus: Possibility to have play/pause/mutt/prev and next button in the screenlocker would be a very useful feature.

Side note: many thanks for this player. This the best player so far most notably thanks to be folder-based. Keep up the good work!

**Comment by souch** [comment]
Yes, why not, I'll have a look someday. I don't see the need of a mute button if there is a play/pause button.
So I would put this button : play/pause - prev - next .

**Comment by drzraf** [comment]
I agree with you about the "mute" button

**Comment by souch** [comment]
ok.
